src/OtherQuantityHandler.cpp

```diff
@@ -177,10 +186,12 @@ void OtherQuantityHandler::DefineQuantities() {
     const len_t n1_re = (this->runawayGrid==nullptr ? 0 : this->runawayGrid->GetMomentumGrid(0)->GetNp1());
     const len_t n2_re = (this->runawayGrid==nullptr ? 0 : this->runawayGrid->GetMomentumGrid(0)->GetNp2());
 
+    const real_t *x = unknowns->GetUnknownData(id_ncold);
+
     // HELPER MACROS (to make definitions more compact)
     // Define on fluid grid
     #define DEF_FL(NAME, DESC, FUNC) \
-        this->all_quantities.push_back(new OtherQuantity((NAME), (DESC), fluidGrid, 1, FVM::FLUXGRIDTYPE_DISTRIBUTION, [this](QuantityData *qd) {FUNC}));
+        this->all_quantities.push_back(new OtherQuantity((NAME), (DESC), fluidGrid, 1, FVM::FLUXGRIDTYPE_DISTRIBUTION, [this,x](QuantityData *qd) {FUNC}));
```

Comment:
Prefer ``&x`` to provide a reference to ``x`` rather than copying the address it's pointing to into a new variable